content/zh-cn/docs/concepts/services-networking/network-policies.md

```diff
@@ -112,7 +112,7 @@ allowed connections will also be implicitly allowed.
 The effects of those `egress` lists combine additively.
 -->
 默认情况下，一个 Pod 的出口是非隔离的，即所有外向连接都是被允许的。如果有任何的 NetworkPolicy
-选择该 Pod 并在其 `policyTypes` 中包含 “Egress”，则该 Pod 是出口隔离的，
+选择该 Pod 并在其 `policyTypes` 中包含 "Egress"，则该 Pod 是出口隔离的，
```

Comment:
why use English quotation instead of Chinese? same as below

Reply:
The characters in the quotation marks are in English. From preview, I think the English quotes seems better.
<img width="615" alt="image" src="https://github.com/kubernetes/website/assets/76980726/91a24ffa-bcdb-497c-ae4d-1b1f27e42fce">
<img width="570" alt="image" src="https://github.com/kubernetes/website/assets/76980726/2fb96553-87e2-4dff-981b-0d88707d39a0">
What do you think?

Reply:
cc @tengqm

Reply:
I don't have a strong opinion on this.